assets/static/js/buzzer.js

```diff
@@ -264,6 +279,18 @@ function checkIfAddPartyFormComplete() {
   }
 }
 
+//function to handle when user updates party size in table.
+function updatePartySize(){
```

Comment:
Can you rename this to `registerUpdatePartySizeClickHandler` to conform to the other click handler naming convention